Makefile

```diff
@@ -56,6 +56,10 @@ SYSCONFDIR:=$(DEFAULT_SYSCONFDIR)
 endif
 SYSCONFDIR:=$(subst PREFIX,$(PREFIX),$(SYSCONFDIR))
 
+ifndef SYSTEMDDIR
```

Comment:
Good to know about this syntax, thanks.  Note that lines before this all use "ifndef" statements.